File: EndGate/EndGate.Core.Client.JS/Loopers/LooperCallback.js
var EndGate;
(function (EndGate) {
    (function (Core) {
        (function (Loopers) {
            var LooperCallback = (function () {
                function LooperCallback(callback) {
                    this._type = "LooperCallback";
                    this.Callback = callback;
                    this.ID = LooperCallback._ids++;
                }
                LooperCallback._ids = 0;
                return LooperCallback;
            })();
            Loopers.LooperCallback = LooperCallback;            
        })(Core.Loopers || (Core.Loopers = {}));
        var Loopers = Core.Loopers;
    })(EndGate.Core || (EndGate.Core = {}));
    var Core = EndGate.Core;
})(EndGate || (EndGate = {}));

File: EndGate/EndGate.Core.Net/Loopers/LooperCallback.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EndGate.Core.Net.Loopers
{
    internal class LooperCallback
    {
        public LooperCallback()
        {
        }

        public LooperCallback(Action callback)
        {
            Callback = callback;
        }

        public Action Callback { get; set; }        
    }
}
